unity-client/Assets/Scripts/MainScripts/DCL/Controllers/MapRenderer/GlobalUsersPositionMarker/Handlers/UserPositionHandler.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using UnityEngine;
+
+internal class UserPositionHandler : IDisposable
```

Comment:
Could we add a documentation header here to explain what is the responsibility of this class? Thanks!